pkg/apis/kubernikus/v1/secret.go

```diff
@@ -16,6 +16,9 @@ type Secret struct {
 	DexClientSecret   string `json:"dex-client-secret,omitempty"`
 	DexStaticPassword string `json:"dex-static-password,omitempty"`
 
+	AuditUserDomainID    string `json:"audit-user-domain-id,omitempty"`
```

Comment:
I'm wondering why this is prefixed with `Audit`. Isn't this the domain_id of the kubernikus service user and the domain_id of the hosting project? So this would be rather generic and should be called `UserDomainID` and `ProjectDomainID`?